packages/lyra/src/lyra.ts

```diff
@@ -25,6 +25,7 @@ export type SearchParams = {
   properties?: "*" | string[];
   limit?: number;
   offset?: number;
+  exact?: boolean | undefined;
```

Comment:
We could get rid of `| undefined`, if the value is not defined will be interpreted as `false` by default